app/models/create_post.rb

```diff
@@ -0,0 +1,15 @@
+# create_posts table
+class CreatePost < ApplicationRecord
+  has_many :create_comments, foreign_key: :create_posts_id
+  has_many :create_likes, foreign_key: :create_posts_id
+  belongs_to :author, class_name: 'CreateUser', foreign_key: :author_id
+  after_save :update_posts_counter
+
+  def latest_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_posts_counter
+    author.increment(:posts_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] It will also be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to the database, then make the update_post_counter method private

 ```
 after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end

```